CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ The public API of this library consists of the functions declared in file
 - Normalize output of h3SetToMultiPolygon to align with the GeoJSON spec, ensuring that each polygon has only one outer loop, followed by holes (#131)
 ### Changed
 - Longitude outputs are now guaranteed to be in the range [-Pi, Pi]. (#93)
+### Updated
```

Comment:
Nit: This should just go under `Changed` I think.

Reply:
Well I put it under Updated because this mathematically does not change the output for k >= 0, but will put it under Changed.